src/components/ListLayout.jsx

```diff
@@ -98,25 +163,47 @@ const ListLayout = ({ items, localToken }) => {
           <ImCross />
         </button>
       </div>
-      <ul className="grid grid-cols-2 justify-around">
-        {items
-          .filter((item) => item.id.includes(filter.toLowerCase()))
-          .map((item, idx) => (
-            <li className="flex flex-col my-4" key={idx}>
-              <div>
-                {` Item Name: ${item.itemName}`}{' '}
-                <input
-                  type="checkbox"
-                  checked={within24hours(item.purchasedDate)}
-                  onChange={(e) => handleCheckboxChange(e)}
-                  name={item.id}
-                  aria-label={item.itemName} //what do we want to call this ('item', 'item.itemName', 'purchased item' .....)
-                />
-              </div>
-              <div>{` Frequency: ${item.previousEstimate}`}</div>
-            </li>
-          ))}
-      </ul>
+      {
+        // have attempted some logic to hide the group if there are no items in that group
```

Comment:
Love work-in-progress notes - never feel shy to add those, and more details the better